exa/__init__.py

```diff
@@ -20,41 +21,64 @@ class Base(HasTraits):
     traits and trait-based logic.
     """
 
+    @staticmethod
+    def right_now():
+        """Returns the current datetime"""
+        return dt.datetime.now()
+
+    @staticmethod
+    def time_diff(start):
+        """Returns a formatted string of the time difference
+        between right now and the passed in datetime"""
+        stop = dt.datetime.now()
+        return '{:.2f}s'.format((stop - start).total_seconds())
+
     @property
     def log(self):
+        """A configured `logger` object"""
         name = '.'.join([
             self.__module__, self.__class__.__name__
         ])
         return logging.getLogger(name)
 
     @classmethod
     def from_yml(cls, path):
+        """Load an object from a configuration file"""
         return cls(**cls._from_yml(path))
 
     @staticmethod
     def _from_yml(path):
+        """Load a configuration file"""
         with open(path, 'r') as f:
             cfg = yaml.safe_load(f.read())
         return cfg
 
     def traits(self, *args, **kws):
+        # inherit super.__doc__?
         # inherent to traitlets API and
         # of little concern to us here.
         skipme = ['parent', 'config']
         traits = super().traits(*args, **kws)
         return {k: v for k, v in traits.items()
                 if k not in skipme}
 
-    def trait_items(self):
-        return {k: getattr(self, k)
-                for k in self.traits()}
+    def trait_values(self):
```

Comment:
You are right, that is why it was originally called items. I think what I was aiming for was to turn traits() into traits_items() and then have traits_values() return the getattr(self, k) but I never followed through on it. Since traitlets already provides traits() it is not worth the extra code.